roles/django_app_k8s/templates/rabbitmq.yml.j2

```diff
@@ -0,0 +1,69 @@
+---
+
+kind: Deployment
+apiVersion: v1
+metadata:
+  labels:
+    app.kubernetes.io/name: rabbitmq
+    app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+    app.kubernetes.io/version: "3.7"
+    app.kubernetes.io/component: async-workers
+    app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+    app.kubernetes.io/managed-by: Ansible
+spec:
+  # TODO: cluster/HA?
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: rabbitmq
+      app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+      app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+  template:
+    metadata:
+      name: opennotificaties-worker
+      labels:
+        app.kubernetes.io/name: rabbitmq
+        app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+        app.kubernetes.io/version: "3.7"
+        app.kubernetes.io/component: async-workers
+        app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+        app.kubernetes.io/managed-by: Ansible
+{% for label in django_app_k8s_extra_rabbitmq_labels %}
+        {{ label }}: {{ django_app_k8s_extra_rabbitmq_labels[label] }}
+{% endfor %}                
+    spec:
+      containers:
+      - name: rabbitmq
+        image: rabbitmq:3.7
+        imagePullPolicy: Always
+
+        resources:
+          requests: {{ django_app_k8s_rabbitmq_resource_requests }}
+          limits: {{ django_app_k8s_rabbitmq_resource_limits }}
+
+        readinessProbe:
+          tcpSocket:
+            port: 5672
+          initialDelaySeconds: 5
+          periodSeconds: 10
+
+        livenessProbe:
+          tcpSocket:
+            port: 5672
+          initialDelaySeconds: 15
+          periodSeconds: 20
```

Comment:
I've seen RabbitMQ go down because of memory watermarks - if 90% of available memory is used, the channels are closed and it basically becomes inoperable. Maybe we can see if we can make this probe better so that we get hard crashes and monitoring starts screaming if it becomes unavailable? That could be for a separate PR though.

Reply:
Ok, separate PR it is.